src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurRequestExtractFileValidationServiceImpl.java

```diff
@@ -40,23 +41,24 @@ public boolean requestExtractHeaderRowValidatesToFileContents(ConcurRequestExtra
         boolean headerValidationPassed;
         LOG.info("requestExtractHeaderRowValidatesToFileContents:  batchDate=" + requestExtractFile.getBatchDate() + "=   recordCount=" + requestExtractFile.getRecordCount() + "=     totalApprovedAmount=" + requestExtractFile.getTotalApprovedAmount() + "=");
         headerValidationPassed = (fileRowCountMatchesHeaderRowCount(requestExtractFile, reportData) &&
-                                  fileOnlyContainsOurEmployeeCustomerIndicator(requestExtractFile, reportData) &&
+                                  fileContainsRequestDetailLines(requestExtractFile, reportData) &&
                                   fileTotalApprovedAmountsAggregatedByRequestIdMatchHeaderTotalApprovedAmount(requestExtractFile, reportData));
         LOG.info("requestExtractHeaderRowValidatesToFileContents:  Header validation : " + ((headerValidationPassed) ? "PASSED" : "FAILED"));
         return headerValidationPassed;
     }
 
     public void performRequestDetailLineValidation(ConcurRequestExtractRequestDetailFileLine detailFileLine, List<String> uniqueRequestIdsInFile) {
-        if (requestDetailFileLineIsCashAdvance(detailFileLine)) {
-            boolean lineValidationPassed = true;
-            lineValidationPassed =  requestedCashAdvanceHasNotBeenClonedInFile(detailFileLine, uniqueRequestIdsInFile);
+        boolean lineValidationPassed = requestedDetailLineContainsValidEmployeeGroupId(detailFileLine);
+        if (lineValidationPassed) { 
```

Comment:
When an invalid employee group id is detected for a row in the file, that line is marked as invalid so that it is not processed. Since that data row is invalid based upon that high level error condition, the rest of the checks do not need to be performed because processing will be aborted later on when the employee group id check is performed in method isDetailFileLineValidCashAdvanceRequest of service implementation ConcurCashAdvancePdpFeedFileServiceImpl .  Refer to lines 66 and 174 of that service implementation.  This is reducing the amount of unnecessary processing and failing the line at the highest error condition.